cmd/gather.go

```diff
@@ -35,13 +36,26 @@ var gatherCmd = &cobra.Command{
 			os.Exit(1)
 		}
 
+		if argStatic && argDynamic {
+			log.Fatalf("Cannot use both 'static' and 'dynamic' only gatherers - exiting")
+			os.Exit(1)
+		}
+
+		gathererTypeArg := ""
```

Comment:
I'm wondering if we shouldn't just assume the default to be "static" and have that be the fallback behaviour? Just a thought.

Reply:
Yep - thought the same beforehand. Default is now static